components/pager/overview.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Pager
+page_title: Pager for Blazor Overview
+description: Add a page navigation to Blazor application
+slug: components-pager-overview
+tags: telerik,blazor,pager,paging
+published: True
+position: 20
+---
+
+# Pager Overview
+
+The `Pager` component will enable you to add page navigation in your Blazor application. It follows the design patter of our components but as standalone one can be used outside of Grid.
+
+To use Telerik Pager component for Blazor:
+1. Add the `TelerikPager` tag
+1. (optional) Populate it's `Total` parameter
+1. (optional) Set it's `ButtonCount` parameter
+1. (optional) Set the `PageChanged` parameter
+1. (optional) Set `Page` or `@bind-Page` (one and two-way data binding)
+1. (optional) Set a `PageSize`
+
+>caption Basic setup of Pager component in your application
+
+````CSHTML
+@*Basic Pager configuration.*@
+
+<TelerikPager Total="TotalItems"
+              ButtonCount="ButtonCount"
+              PageSize="ItemsOnPage"
+              @bind-Page="CurrentPage">
+
+</TelerikPager>
+
+@code {
+    public int TotalItems { get; set; } = 80;
+    public int ButtonCount { get; set; } = 4;
+    public int ItemsOnPage { get; set; } = 10;
+    public int CurrentPage { get; set; } = 2;
+}
+````
+
+>caption The result from the code snippet above
+
+![basic configuration of the pager](images/pager-basic-configuration-screenshot.jpg)
+
+## Features
+* `Class` - The CSS class that will be rendered on the main wrapping element of the Pager.
+* `Total` - **int** - Represents the total count of items in the pager.
```

Comment:
The types of the parameters must also be in code blocks, not bold.

Reply:
Done